jailer.go

```diff
@@ -0,0 +1,388 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package firecracker
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+)
+
+const (
+	defaultJailerPath = "/srv/jailer/firecracker"
```

Comment:
I don't really think I like `/srv`; I'm not sure it really fits into the [standard definition](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard).  What about `/var/lib/jailer/firecracker`?

@nmeyerhans Do you have thoughts on this?

Reply:
The SDK isn't setting that. If not chroot base directory was provided when executing the jailer, it'll go to `/srv/jailer/firecracker/`.

We can optionally have our own default, but that would mean we would always be setting the chroot base directory.

Reply:
This default comes from the jailer itself?  If it comes from the jailer itself, then I don't think we should override the default.

Reply:
Cool, yea, from the jailer itself. I added a comment to make that more clear.